xfs/parse.go

```diff
@@ -394,7 +398,14 @@ func quotaManagerStats(us []uint32) (QuotaManagerStats, error) {
 		Wants:         us[5],
 		ShakeReclaims: us[6],
 		InactReclaims: us[7],
-	}, nil
+	}
+
+	if l == 8 {
+		return s, nil
+	}
```

Comment:
I think here you could make this slightly simpler with this:
```golang
if l > 8 {
    s.Unused = us[8]
}
```
And then just `return s, nil`.  Saves one line of code anyway.

Reply:
Yeah, good point. I just updated the pull request with that style change